pkg/addons/data/Deployment.apps/default/router.yaml

```diff
@@ -48,6 +48,10 @@ spec:
           value: "1936"
         - name: STATS_USERNAME
           value: admin
+        - name: ROUTER_THREADS
+          value: "0"
```

Comment:
what does this do?

Reply:
Performance tweaking for the router. The default is 0. But we can increase based on box sizes, CPU pinning, and enable routers to serve more routes